cli/script/command-executor.ts

```diff
@@ -1175,7 +1175,9 @@ export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void>
                 throw new Error("Platform must be either \"ios\" or \"android\".");
             }
 
-            var cordovaCommand: string = command.build ? "build" : "prepare";
+            var cordovaCommand: string = command.build ? 
+                (command.isReleaseBuildType ? "build --release" : "build") :
```

Comment:
We normally name the flags that we pass through to the underlying `react-native` or `cordova` CLI's the same as they would be in the original CLI. This generally makes them more discoverable/consistent for people who already have been using the underlying CLI. I understand why it might have made sense to name it otherwise, but with all things considered do you think it makes sense to just call the flag `release` instead of `isReleaseBuildType`? (I'll leave this up to you).